db/schema.rb

```diff
@@ -312,13 +220,13 @@
     t.boolean  "poe",                                                 default: false
     t.boolean  "eight_keys",                                          default: false
     t.boolean  "dodmou",                                              default: false
-    t.boolean  "sec_702",                                             default: false
+    t.boolean  "sec_702"
```

Comment:
Is there a reason this one is losing its `default: false`? Should it not be `default: false, null: false`?